server/backend/ios_notifications/ios_apns/iosAPNsRepository.go

```diff
@@ -0,0 +1,143 @@
+package ios_apns
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/TUM-Dev/Campus-Backend/server/backend/ios_notifications/ios_apns/ios_apns_jwt"
+	"github.com/TUM-Dev/Campus-Backend/server/env"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/http2"
+	"gorm.io/gorm"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+const (
+	BundleId          = "de.tum.tca"
+	ReadIdleTimeout   = 15 * time.Second
+	HTTPClientTimeout = 60 * time.Second
+)
+
+const (
+	APNsDevelopmentURL = "https://api.sandbox.push.apple.com:443"
+	APNsProductionURL  = "https://api.push.apple.com:443"
+)
+
+var (
+	ErrCouldNotSendNotification   = errors.New("could not send notification")
+	ErrCouldNotDecodeAPNsResponse = errors.New("could not decode apns response")
+)
+
+type Repository struct {
+	DB         gorm.DB
+	Token      *ios_apns_jwt.Token
+	httpClient *http.Client
+}
+
+// APNsURL uses the environment variable ENVIRONMENT to determine whether
+// to use the production or development APNs URL.
+func (r *Repository) APNsURL() string {
+	if env.IsProd() {
+		return APNsProductionURL
+	}
+
+	return APNsDevelopmentURL
+}
+
+// CreateCampusTokenRequest creates a request log in the database that can be referred to
```

Comment:
```suggestion
// Creates a request log in the database that can be referred to
```

Reply:
![Screenshot 2023-01-23 at 12 13 15](https://user-images.githubusercontent.com/17065257/214026053-9668e843-1e1c-45be-aec3-648b89f3cd9e.png)

Reply:
The standard lib also uses this style

Reply:
OK, then never mind. I did not knew that one!